modules/core/ante/ante.go

```diff
@@ -8,20 +8,20 @@ import (
 	"github.com/cosmos/ibc-go/v4/modules/core/keeper"
 )
 
-type AnteDecorator struct {
+type RedundancyDecorator struct {
```

Comment:
Thoughts on `RedundantRelayDecorator`?

Reply:
I like @colin-axner's suggestion of `RedundantRelayDecorator`. It's probably a little more descriptive :)